frontend/src/store/questions/constants.js

```diff
@@ -1,23 +1,27 @@
 import {
   getQuestions,
   getQuestion,
-  getAnswers
+  getAnswers,
+  postAnswer
 } from '@/utils/api/questions'
 
 export const ACTIONS = {
```

Comment:
Just a note on these symbols. They can (and probably should) be more descriptive as they're just unique key hashes but in debuggint they will give more relevant information. That was my bad for starting the convention.